app/Http/Controllers/Frontend/HomePageController.php

```diff
@@ -64,22 +67,39 @@ public function getNews($id)
     {
     	$categories = $this->categoryModel->all();
         $detail = $this->detailModel->find($id);
-        $tags = $this->tagModel->orderBy('created_at', 'desc')->get()->take(15);
     	$details = $this->detailModel->all();
         $relatedDetail = $this->detailModel->where('category_id', '=', $detail->category->id)
             ->where('id', '!=', $detail->id)
             ->orderBy('created_at', 'desc')->get()->take(4);
-        $tag = $this->tagModel->find($id);
+        $tags = $this->tagModel->orderBy('created_at', 'desc')->get()->take(15);
         $tagsDetail = $this->tagModel->where('detail_id', '=', $detail->id )->get();
+        $commentsDetail = $this->commentModel->where('detail_id', '=', $detail->id )->get();
```

Comment:
chỗ này cũng vậy, bỏ dấu '=' đi chỉ cần $this->commentModel->where('detail_id', $detail->id )->get();